scripts/clean_skops.py

```diff
@@ -4,16 +4,31 @@
 removed.
 """
 
+import datetime
+
+import pandas as pd
 from huggingface_hub import HfApi
 
 # This is the token for the skops user. TODO remove eventually, see issue #47
 token = "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"
 client = HfApi()
 user = client.whoami(token=token)["name"]
-answer = input(f"Are you sure you want to delete all repos under {user}? (y/[n])")
+answer = input(
+    f"Are you sure you want to delete all repos under {user} older than 7 days? (y/[n])"
+)
 if answer != "y":
     exit(1)
 models = client.list_models(author=user)
+
+print(f"Found {len(models)} models, checking their age...")
+
 for model_info in models:
-    print(f"deleting {model_info.modelId}")
+    info = client.model_info(model_info.modelId, token=token)
+    age = (
+        datetime.datetime.now(datetime.timezone.utc) - pd.to_datetime(info.lastModified)
```

Comment:
If you don't want pandas to run the script, you could parse the time info as:

`datetime.datetime.fromisoformat(info.lastModified.rsplit('.', 1)[0])`

Reply:
That actually reminded me that I was missing the `huggingface_hub` dependency here. Added that.

I changed to your version, but it's not the same as before, since it's now ignoring timezone info. Not a big deal in this case though.

Happy for this to be merged.

Reply:
Yeah, we could add the tz back but it's not that important.